test/unit/config.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+const config = require('../../app/lib/config');
+
+const expect = chai.expect;
+
+const disableScheduler = process.env.DISABLE_SCHEDULER;
+
+function resetEnv() {
+  process.env.disableScheduler = disableScheduler;
+}
+
+describe('config', () => {
+  describe('getRule', () => {
```

Comment:
Is it worth having a test to check the schedule is set to `process.env.ETL_SCHEDULE` when it is set, rather than the default?